### ISSUE TITLE
Login error: JSON decode fail, 403 - Forbidden

### ISSUE THREAD

**Issue by mktTIargobrazil** [instaloader/instaloader#2126]
I can't use the instaloader anymore, it's still appear this error: **Login error: JSON decode fail, 403 - Forbidden**. I check the code, but nothing work, Can you help me?

**Comment by WillYouth** [comment]
Hi I faced the same issue too

**Comment by Sam-bit** [comment]
I am still facing the same error. the latest user agent as per [https://www.whatismybrowser.com/guides/the-latest-user-agent/chrome](https://www.whatismybrowser.com/guides/the-latest-user-agent/chrome) is
**Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36**

**Comment by mktTIargobrazil** [comment]
https://instaloader.github.io/troubleshooting.html#login-error
Try this code, it's can solve the problem!

**Comment by marco-land** [comment]
> https://instaloader.github.io/troubleshooting.html#login-error Try this code, it's can solve the problem!

I'm on a Mac and am getting this error when running the script:
```
Instaloader not found.
  pip install [--user] instaloader
```

I am using pip3, not sure if this is related in any way

**Comment by marco-land** [comment]
> https://instaloader.github.io/troubleshooting.html#login-error Try this code, it's can solve the problem!

I get this error:
```
Instaloader not found.
  pip install [--user] instaloader
```

instaloader is installed though

**Comment by ericdoh0401** [comment]
What exactly is this code supposed to help with? If I do not have access to Firefox, what can I do to solve this issue? My current code throws the "instaloader.exceptions.ConnectionException: Login error: JSON decode fail, 403 - Forbidden." However, a couple of weeks ago, it worked fine.

**Comment by aandergr** [comment]
Fixed in [Version 4.10.2](https://github.com/instaloader/instaloader/releases/tag/v4.10.2) – thanks @LostXOR !
